<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchGroupBox</class>
 <widget class="QGroupBox" name="SwitchGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>67</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkToggle">
     <property name="text">
      <string>Toggle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelButton">
     <property name="text">
      <string>Button:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonState">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
